tests/src/com/unciv/testing/TutorialMinerTests.kt

```diff
@@ -0,0 +1,54 @@
+package com.unciv.testing
+
+import com.unciv.testing.mocks.JsonParserMock
+import com.unciv.ui.tutorials.TutorialMiner
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(GdxTestRunner::class)
+class TutorialMinerTests {
+
+    private val languages = listOf(
+            "English", "Czech", "French", "Italian",
+            "Korean", "Polish", "Ukrainian", "Russian",
+            "Simplified_Chinese", "Traditional_Chinese"
+    )
+    private val jsonParser = JsonParserMock()
+    private val tutorialMiner = TutorialMiner(jsonParser)
+
+    @Test
+    fun getCivilopediaTutorials() {
+        // GIVEN
+        val expectedTutorKeys = listOf(
```

Comment:
Why not take this directly from the tutorial Enum you created? That way when adding a tutorial there will be one source of truth and the test will be synced to it